tests/integration/test_race_conditions.py

```diff
@@ -16,7 +16,7 @@ def setUp(self):
         pass
 
 
-DELAY_TIME = 1
+DELAY_TIME = 0.2
```

Comment:
heh that in itself should be a huge speedup. Should be just fine - if for some reason that's ever not a long enough delay to actually have the responses arrive at the front end in the desired order, it would be a (presumably rare) missed test, or testing a *different* order, but one that should still be valid. So I'm happy to make this change, but some day we should refactor this test to something a little smarter that actually triggers the responses based on evidence that the previous response has been received and acted upon, rather than just time.